test_ast.py

```diff
@@ -1060,6 +1061,35 @@ def test_system_include(self):
         self.assertEqual(1, len(nodes), repr(nodes))
         self.assertEqual(Include('vector', system=True), nodes[0])
 
+    def test_system_include(self):
```

Comment:
It looks like `test_system_include()` shadows the `test_system_include()` that is defined a few lines above.